Python/pylifecycle.c

```diff
@@ -751,6 +752,11 @@ pycore_init_types(PyInterpreterState *interp)
     return _PyStatus_OK();
 }
 
+static const char INTERPRETER_TRAMPOLINE_CODE[] = {
```

Comment:
```suggestion
static const uint8_t INTERPRETER_TRAMPOLINE_CODE[] = {
```

I think we should probably be better about using explicit, unsigned values for bytecode, since I'm pretty sure that the signed overflow from assigning an opcode/oparg with value >= 128 is technically undefined.

Also, consider lowercase for `INTERPRETER_TRAMPOLINE_CODE`? I get that it's a constant, but our style typically reserves uppers for macros.

Reply:
We use ALL_CAPS for constants as well.